ADAL/src/request/ADWebRequest.m

```diff
@@ -153,6 +156,7 @@ - (void)send
     }
     
     NSURL* requestURL = [ADHelpers addClientVersionToURL:_requestURL];
+    requestURL = [[ADAuthorityValidation sharedInstance] networkUrlForAuthority:requestURL context:self];
```

Comment:
What if it's an  ADFS authority? I assume the result won't be found and original authority will be returned, but I still feel a little bit wrong checking aadCache in any case.

Reply:
👍